pic-sure-hpds-ui/src/main/webapp/picsureui/common/router.js

```diff
@@ -1,18 +1,39 @@
-define(["jquery", "backbone", "auth/login", "common/queryBuilderView"],
-        function($, Backbone, login, queryBuilderView){
-    var Router = Backbone.Router.extend({
+define(["common/searchParser", "backbone", "common/session", "login/login", 'header/header', 'footer/footer','user/userManagement',
```

Comment:
The previous version of this file was not being used so you can ignore the diff and just look at the latest file. Essentially, this is the psamaui router.js file, with the query builder added, and a mechanism to override and add routes